app/views/foreman_wreckingball/hosts/schedule_remediate.html.erb

```diff
@@ -0,0 +1,35 @@
+<% title _('Remediate %s') % @host %>
+<% javascript 'foreman_tasks/trigger_form' %>
+<% stylesheet 'foreman_tasks/trigger_form' %>
+
+
+<% if @status.class.respond_to?(:dangerous_remediate?) && @status.class.dangerous_remediate? -%>
+<%= alert(:text => _('This will cause a service interruption.'), :class => 'alert-warning', :close => false) %>
+<% end -%>
+
+<%= form_for @triggering, :url => submit_remediate_host_path(:id => @host.id, :status_id => @status), :html => {:id => 'schedule_remediate_form'} do |f| %>
```

Comment:
Can you remove the `well` css class from the form please? You have to explicitly set `:class=>"form-horizontal well"`.
https://github.com/theforeman/foreman/blob/b03dcd1b7ebee341ad5090644147d37ad1c3eafa/app/helpers/form_helper.rb#L292